src/components/05_pages/Permissions/Permissions.jsx

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
 import { css } from 'emotion';
 
-const Permissions = () => (
-  <div>
-    <h1 className={styles.title}>Permissions</h1>
-    <p>This will be the permissions page.</p>
-  </div>
-);
+const drupalOrigin = 'http://localhost:9998';
```

Comment:
let's add this in with dotenv (already included in CRA)

Reply:
There is a bit of a pane involved with CRE here. Environment variables aren't passed along unless they are prefixed with react. I had this problem when working on my config experiment, see https://github.com/jsdrupal/drupal-admin-ui/compare/config-experiment-2#diff-01454d5c4c96a618ac33dd690eb3c20bR128

Reply:
I hope we can workaround that though :)

Reply:
oh yeah, I forgot about that. I think prefixing with REACT_ is totally fine :+1: